app/build.gradle

```diff
@@ -1,72 +0,0 @@
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
-def version = "0.11.0"
-
-android {
-    compileSdkVersion 28
-    buildToolsVersion "28.0.3"
-
-    compileOptions {
-        sourceCompatibility 1.8
-        targetCompatibility 1.8
-    }
-
-    defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 28
-        versionCode 905
-        versionName version
-    }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
-        debug {
-            applicationIdSuffix ".debug"
-            debuggable true
```

Comment:
Was this redundant?

Reply:
The "debug" version has debug on by default. I think there's some more redundant settings but I didn't look up everything.

> When you create a new module, Android Studio automatically creates the debug and release build types for you. Although the debug build type doesn't appear in the build configuration file, Android Studio configures it with debuggable true. 

[source](https://developer.android.com/studio/build/build-variants)